### DISCUSSION TITLE
Could not setup phive

### DISCUSSION THREAD

**Discussion by MGatner** [shivammathur/setup-php#540] [Q&A / Help] [answered]
Just started happening tonight, no other details in the logs. I've tried rerunning the job multiple times.

An example of the failed run: https://github.com/tattersoftware/codeigniter4-frontend/runs/4500359540?check_suite_focus=true

Here's an excerpt:
```bash
Run shivammathur/setup-php@v2
/usr/bin/bash /opt/hostedtoolcache/linux.sh 8.0 /home/runner/work/_actions/shivammathur/setup-php/v2/dist

==> Setup PHP
✓ PHP Found PHP 8.0.13

==> Setup Tools
✓ composer Added composer 2.1.14
✓ PECL Added PECL 1.10.12
chmod: cannot access '/usr/local/bin/phive': No such file or directory
✗ phive Could not setup phive

==> Setup Extensions
✓ intl Enabled
✓ json Enabled
✓ mbstring Enabled
✓ xml Enabled

==> Sponsor setup-php
✓ setup-php https://setup-php.com/sponsor
```

Relevant portion of the YAML:
```yaml
jobs:
  build:
    name: Architectural Inspection
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v2

      - name: Setup PHP
        uses: shivammathur/setup-php@v2
        with:
          php-version: '8.0'
          tools: composer, pecl, phive
          extensions: intl, json, mbstring, xml
        env:
          COMPOSER_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

**Comment by shivammathur** [comment, marked as answer]
Fetching https://phar.io/releases/phive.phar is failing because of SSL issues. Please report this [upstream](https://github.com/phar-io/phive)

**Reply by MGatner** [reply]
Thanks! Will do.

**Reply by shivammathur** [reply]
I changed the source for phive to GitHub releases. It should work now.

**Reply by MGatner** [reply]
https://github.com/phar-io/phar.io/issues/113

**Reply by MGatner** [reply]
They have fixed the issue, but it sounds like their site link was already just a redirect so you may want to stick with GitHub anyway.
